.travis.yml

```diff
@@ -16,9 +20,13 @@ jobs:
   exclude:
     - os: osx
       env: CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake" # don't test MinGW from OSX
+    - os: linux
+      env: CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake"
```

Comment:
A clarifying comment as in line 22 would be nice here.

Reply:
Thanks @bk138 , I have added the comment.